src/devices/holman_ws5029.c

```diff
@@ -141,29 +211,61 @@ r_device const holman_ws5029pcm = {
         .fields      = output_fields,
 };
 
-/**
+/** int holman_ws5029pwm_decode(r_device *decoder, bitbuffer_t *bitbuffer)
```

Comment:
A function link is `/** @fn int holman…`

Reply:
Done, thx